### DISCUSSION TITLE
When calling uws.publish, get warning

### DISCUSSION THREAD

**Discussion by car1ot** [uNetworking/uWebSockets.js#250] [Q&A] [answered]
Sometimes get this warning, when publishing message:

`Invalid access of closed uWS.WebSocket/SSLWebSocket.
(node:7688) UnhandledPromiseRejectionWarning: Invalid access of closed uWS.WebSocket/SSLWebSocket.
(node:7688) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)`

**Comment by yovanoc** [comment]
Same for me

**Comment by ghost** [comment, marked as answer]
That's not a warning its a message you have fundamental flaws in your code. Read the docs and follow them.

**Comment by yovanoc** [comment]
No, he's right. I will do an example repo. Only with app.publish we can get that warning, so, nothing can be wrong in our part.

**Comment by ghost** [comment]
There is no function called uws.publish and app.publish doesn't give any messages

**Comment by yovanoc** [comment]
I found a case where it give exactly this message, I will make a repo soon.

**Comment by ghost** [comment]
There is no function called uws.publish and app.publish doesn't throw any such exceptions:

https://github.com/uNetworking/uWebSockets.js/blob/master/src/AppWrapper.h#L245

Closing until proven otherwise.
